README.md

```diff
@@ -173,10 +186,10 @@ and the
 
 So that will require:
  * Apache Maven 3.3.1 or later.
- * Java 8 JDK
+ * Java 11 JDK (https://adoptopenjdk.net)
```

Comment:
AdoptOpenJDK has moved under the Eclipse Foundation and has been rebranded: https://adoptium.net/faq.html#AdoptOpenJDK

Reply:
I suggest to remove the specific bracketed text.